File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.java
class Solution {
    public String solution(String s, int n) {
        StringBuffer answer = new StringBuffer();
        char[] ch = s.toCharArray();
        for(char c : ch){
            if(c>='A' && c<='Z'){
                c = (char)((c+n-'A')%26 +'A');
            }else if(c>='a' && c<='z'){
                c = (char)((c+n-'a')%26 +'a');
            }
            answer.append(c);
        }
        return answer.toString();
    }
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    var answer = '';
    return [...s]
        .map(ch => {
            let code = ch.charCodeAt(0);
            if(65 <= code && code <= 90) {
                code = code+n > 90 ? code+n-26 : code+n;
            }
            else if(97 <= code) {
                code = code+n > 122 ? code+n-26 : code+n;
            }
            return String.fromCharCode(code)
        })
        .join('');
}